### PR TITLE
Polish LobbyComponent CSS

### PR DESCRIPTION
**Pull request by tirr-c**

### CODE REVIEW

File: frontend/src/app/lobby.component.html [hunk 1]
```diff
@@ -1,44 +1,37 @@
-<div class="mdl-grid">
-  <div class="mdl-layout-spacer"></div>
-  <div class="mdl-cell mdl-cell--6-col-desktop mdl-cell--8-col-tablet mdl-cell--4-col-phone">
-    <h4 class="mdl-color-text--red" *ngIf="error | async" style="display: inline">{{error | async}}</h4>
-    <button class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--accent" (click)="gotoCreateGame()" style="float: right">
+<div class="lobby">
+  <div class="room-header">
+    <div *ngIf="loading | async" class="spinner mdl-spinner mdl-js-spinner is-active"></div>
+    <h4 *ngIf="error | async" class="error mdl-color-text--red">{{error | async}}</h4>
```
Comment (edwardycl) [inline frontend/src/app/lobby.component.html]: 다른 component들에서는 .error이 아니라 .error-message임

File: frontend/src/app/lobby.component.html [hunk 2]
```diff
@@ -1,44 +1,37 @@
...
+    <div class="mdl-layout-spacer"></div>
+    <button class="new-game mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--accent" (click)="gotoCreateGame()">
```
Comment (edwardycl) [inline frontend/src/app/lobby.component.html]: .new-game-button 어떨까

**Comment by edwardycl** [comment]
@web-mighty-homu r+

**Comment by web-mighty-homu** [comment]
:pushpin: Commit 7f00356 has been approved by `edwardycl`

**Comment by web-mighty-homu** [comment]
:hourglass: Testing commit 7f0035639843fb30eedde12927c606d52d8e2e3f with merge f9b28fd3ffd0da1a16ff9565725fc176b180e03f...

**Comment by web-mighty-homu** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/swsnu/swpp17-team2/builds/311168799?utm_source=github_status&utm_medium=notification)
Approved by: edwardycl
Pushing f9b28fd3ffd0da1a16ff9565725fc176b180e03f to master...

**Pull request merged**